File: Microservices/Rethink.Services.Common/Enums/Billing/PatientInvoiceStatus.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Rethink.Services.Common.Enums.Billing
{
    public enum PatientInvoiceStatus
    {
        [Description("Ready To Invoice")]
        ReadytoInvoice = 1,
        [Description("Invoice Sent")]
        InvoiceSent = 2,
        [Description("Partially Paid")]
        PartiallyPaid = 3,
        [Description("Fully Paid")]
        FullyPaid = 4
    }
}

File: billing_ui_app/src/app/core/enums/billing/patient-invoice-status.ts
export enum PatientInvoiceStatus {
  ReadyToInvoice = 1,
  InvoiceSent = 2,
  PartiallyPaid = 3,
  FullyPaid = 4
}

export const PatientInvoiceStatusLabel: Record<number, string> = {
  1: "Ready To Invoice",
  2: "Invoice Sent",
  3: "Partially Paid",
  4: "Fully Paid"
};